src/Value/Bsn.php

```diff
@@ -19,7 +19,9 @@ public static function getConstraints(): array
     }
 
     /**
-     * {@inheritDoc}
+     * @inheritDoc
+     *
+     * @param string $value
```

Comment:
This should be inherited based on the template in AbstractString

Reply:
That is correct, but psalm failed when I didn't add it. This was not the case in 4.0 so it probably is a bug in lower versions.